<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SysexLive</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/IMG/IMG/SysexLive.png</normaloff>:/IMG/IMG/SysexLive.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelSynth1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Synth 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxSynth1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBoxSynth2"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelSynth2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Synth 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Song Registry Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FullName 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FullName 2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Synth 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Synth 2</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddEntry"/>
    <addaction name="actionDeleteEntry"/>
    <addaction name="separator"/>
    <addaction name="actionMoveUp"/>
    <addaction name="actionMoveDown"/>
    <addaction name="separator"/>
    <addaction name="actionSearchInterfaces"/>
    <addaction name="separator"/>
    <addaction name="actionSendPatches"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAboutSysexLive"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSearchInterfaces"/>
   <addaction name="separator"/>
   <addaction name="actionAddEntry"/>
   <addaction name="actionDeleteEntry"/>
   <addaction name="actionMoveUp"/>
   <addaction name="actionMoveDown"/>
   <addaction name="separator"/>
   <addaction name="actionSendPatches"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddEntry">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Window-Add-icon.png</normaloff>:/IMG/IMG/Window-Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>Add Entry</string>
   </property>
  </action>
  <action name="actionDeleteEntry">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Delete-icon.png</normaloff>:/IMG/IMG/Delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/File-New-icon.png</normaloff>:/IMG/IMG/File-New-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Folder-Open-icon.png</normaloff>:/IMG/IMG/Folder-Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Save-icon.png</normaloff>:/IMG/IMG/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Arrow-Up-icon.png</normaloff>:/IMG/IMG/Arrow-Up-icon.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Arrow-Down-icon.png</normaloff>:/IMG/IMG/Arrow-Down-icon.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionAboutSysexLive">
   <property name="text">
    <string>About SysexLive</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSearchInterfaces">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Search-icon.png</normaloff>:/IMG/IMG/Search-icon.png</iconset>
   </property>
   <property name="text">
    <string>Search Interfaces</string>
   </property>
  </action>
  <action name="actionSendPatches">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/IMG/IMG/Mail-icon.png</normaloff>:/IMG/IMG/Mail-icon.png</iconset>
   </property>
   <property name="text">
    <string>Send Patches</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
